.circleci/config.yml

```diff
@@ -91,7 +101,14 @@ aliases:
     command: |
        export PATH=$WORKDIR/miniconda/bin:$PATH
        source activate $PY_VER
-       coveralls
+       # possibly turn off coverage
+       first=$(git cherry master -v | head -n1)
+       current=$(git log -n1 --pretty=oneline)
+       if [[ $first == $current || $current != *"no_coverage"*  ]]; then
```

Comment:
@doutriaux1 Can you provide a quick high-level description of what you're trying to capture here?  I don't think I understand the git workflow you have in mind for turning off coverage with commits.

Reply:
just if you have "no_coverage" in commit and it's not the first commit on the PR then skip coverage.

Reply:
because coverage is slow